runtime/jcl/common/stdinit.c

```diff
@@ -260,6 +260,23 @@ standardInit( J9JavaVM *vm, char *dllName)
 		vm->jlrMethodInvoke = ((J9JNIMethodID *) invokeMethod)->method;
 		(*(JNIEnv*)vmThread)->DeleteLocalRef((JNIEnv*)vmThread, clazz);
 
+#if JAVA_SPEC_VERSION >= 18
+		{
+			jmethodID invokeMethodMH = NULL;
+
+			clazz = (*(JNIEnv*)vmThread)->FindClass((JNIEnv*)vmThread, "java/lang/reflect/Method");
```

Comment:
This class is already found above, can we just reuse it?

Reply:
Fixed. Reused the `clazz` and moved `DeleteLocalRef()`.